.github/CODEOWNERS

```diff
@@ -38,6 +38,7 @@ yarn.lock                                         @backstage/maintainers @backst
 /plugins/devtools                                 @backstage/maintainers @backstage/reviewers @awanlin
 /plugins/devtools-backend                         @backstage/maintainers @backstage/reviewers @awanlin
 /plugins/devtools-common                          @backstage/maintainers @backstage/reviewers @awanlin
+/plugins/dynatrace                                @backstage/maintainers @backstage/reviewers @Dynatrace
```

Comment:
probably we would need to create a new `@backstage/dynatrace-reviewers` team, as I don't think it's possible to specify an entire organization as code owner 🤔